zulip_bots/zulip_bots/run.py

```diff
@@ -73,11 +80,13 @@ def parse_args():
                         help='Install dependencies for the bot.')
     options = parser.parse_args()
 
-    if not options.name and not options.path_to_bot:
+    if (not options.name and not options.path_to_bot) or is_not_equal(options.name, options.path_to_bot):
         error_message = """
 You must either specify the name of an existing bot or
 specify a path to the file (--path-to-bot) that contains
 the bot handler class.
+If you provide both: the name and the path to the bot,
+make sure that both point to the same bot.
```

Comment:
I could have separated out the above 4-5 lines of code in the if block to create a new function, as these lines are being used again in `zulip_bot_output.py` file.
But it doesn't make much sense to me as this error message won't be used anywhere else except for these 2 places.

I guess the modifications to create a new function for a code being used twice isn't worth it. What are your thoughts?

Reply:
I agree, modifiying the function in run.py certainly doesn't hurt.

Reply:
Sure! Makes sense.